### PR TITLE
Update readme for the creation of the WG

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
* Add section on getting started since this is a likely location new participants will want to look
* Remove (inactive?) slack channel
* "Legacy" Gamepad Extensions
* WebVR CG -> Immersive Web CG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,28 @@
...
+| VR | VR Devices, previously handled by WebVR | Magic Window Behaviour     |
+| AR | Mixed Reality Headsets                  | Phone AR                   |
 
-The specification has recently undergone a name change, so expect to see multiple references to "WebVR" sprinkled throughout.
+The [WebXR Device API Specification][1] is the repository of the [Immersive Web Working Group][17]. You can read the [Charter Here][18].
 
+## Taking Part
+
+1. Read the [code of conduct][19]
+2. See if your issue is bring discussed in the [issues][8], or if your idea is being discussed in the [proposals repo][20].
```
Comment (ddorwin) [inline README.md]: typo: bring

s/is being/is or has been discussed/? Note, though, that the default view is only open issues.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,28 @@
...
+3. We will be publishing the minutes from the bi-weekly calls.
```
Comment (ddorwin) [inline README.md]: This seems more about the WG than the spec. Perhaps the spec repo should just link to a page on the WG site.

File: README.md [hunk 3]
```diff
@@ -2,19 +2,28 @@
...
+4. You can also join the working group to participate in these discussions.
 
 ## Specifications
```
Comment (ddorwin) [inline README.md]: There is only one specification in this repo, and it is already linked above. Including all this here may be confusing. Perhaps the rest should be moved to a historical references section or similar. On the other hand, WebVR is already linked above and Gamepad* aren't actually relevant to this spec (at least for now, pending advanced input work).

File: README.md [hunk 4]
```diff
@@ -80,3 +88,7 @@ Per the [`LICENSE.md`](LICENSE.md) file:
 [14]: https://www.w3.org/community/about/agreements/cla/
 [15]: https://www.w3.org/Consortium/Legal/2008/03-bsd-license.html
 [16]: https://immersive-web.github.io/webxr-reference/
+[17]: https://w3.org/immersive-web
```
Comment (ddorwin) [inline README.md]: Remove [9] now that it's unused?

File: README.md [hunk 5]
```diff
@@ -2,19 +2,28 @@
 
 [![Build Status](https://travis-ci.org/immersive-web/webxr.svg?branch=master)](https://travis-ci.org/immersive-web/webxr)
 
-WebXR is driving support for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays on the Web.
+The WebXR device API os for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays on the Web.
```
Comment (ddorwin) [inline README.md]: "sensors" doesn't seem like the right word.

File: README.md (hunk 2 above)
Comment (AdaRoseCannon) [reply README.md]: This is mostly because this is likely to be a first stop for many people looking to get involved.

**Comment by toji** [comment]
Merging now so that the updated readme is available to coincide with the WG announcement. Can address last remaining feeback from David later if needed.

**Pull request merged**